### DISCUSSION TITLE
Has anyone got TMP working with FF release 118.0.1 (Windows 10 22h2)?

### DISCUSSION THREAD

**Discussion by demoergo** [onemen/TabMixPlus#240] [Q&A] [answered]
I'm not looking for support... If I've missed something I'm not looking to waste anyone else's time. I'm just curious if anybody has got it working.
I've been through the instructions/discussions/bug reports with a fine-toothed comb and tried everything I can see to get TMP working again with this new update, but I keep getting ".. cannot be installed because it appears to be corrupt".
Browser toolbox debugger says: tab_mix_plus-dev-build.xpi does not contain a valid manifest(resource://gre/modules/addons/XPIInstall.jsm:685:11) JS Stack trace: loadManifest@XPIInstall.jsm:685:11. 
Contents of the xpi [1.0.0-pre.17] (2023-09-03) looks fine browsing with winrar. I just redownloaded and it has identical hash.
I've tried a creating a new profile and followed all the steps very carefully including changing config.js as per the 115 instructions and setting extensions.experiments.enabled to true etc...  
I'm giving up for now, the only thing can think to try now is uninstalling firefox and starting fresh if it's just me.

**Comment by onemen** [comment, marked as answer]
[Read this](https://github.com/onemen/TabMixPlus/discussions/237#discussioncomment-6915374)

**Comment by demoergo** [comment]
Saw that already. I've gone over everything very carefully multiple times.,. but no luck. Thanks though, I'll try again with a clean install.

**Reply by demoergo** [reply]
Got it working! I figured out what I was doing wrong. I was using xiaoxiaoflood's utils.zip from the main page's installation instructions but I didn't realize the utils_extensions_and_scripts.zip in your post was different. Thank for the help.

**Reply by STaylor135** [reply]
This is actually a big point though. 

_The installation page needs to be updated_


I'm not a developer and really only have a github account to post comments, like this one. I'm not going to fork the project to edit the page and if I had, I would have been wrong. 

I came across this after YEARS of thinking TMP was dead and even run Basillic to have some older plugins and TMP would be one of them.  I found this by rushing away from Chrome and getting plugins and just brain fartted a newer plugin's name and found TMP links, with it's big note that the full plugin was here.  I followed the installation directions for it to not work. When it said that's my fault, do it again.  I found, the config.js file was gone, I know it was there but it got removed.  The [profile]\chorme\utils folder was different. some files aren't even .jsm but .sys.mjs.  Renaming the files don't help.  Also the utils picture doesn't even render correctly in Firefox currently!  Using the files from the installation page, and Firefox nightly, it works with the plugin latest 1.23.  The script here makes it work with the current version of FF 130 (honestly I thought it was 127 something but I did just get an update and can't entirely be sure that maybe the install page scrips work with 130, can't really check because I have more work to do now)

So in short, people will find this project and just not use it because the installation page and linked files are out of date and wrong. 

EDIT: I know this is an old conversation but the focus is exactly the same and demonstrates how this is an old but ongoing issue.

**Reply by pyrates999** [reply]
https://onemen.github.io/tabmixplus-docs/other/installation/

**Reply by STaylor135** [reply]
Yes, that would be the page in question with the image that doesn't show in firefox (but does in chrome) and links to files that work in Nightly but not in release.

**Reply by onemen** [reply]
@STaylor135 

I will appreciate your help to identify the issue you mentioned in your post.
  
If i understand you correctly the `Firefox scripts` files you downloaded from the links [here](https://onemen.github.io/tabmixplus-docs/other/installation/) doesn't work for you in Firefox version 130 but work in Firefox nightly 132.
if that is the case please report about this issue [here](https://github.com/xiaoxiaoflood/firefox-scripts/issues)

Regarding `image that doesn't show in firefox` can you provide a screenshots of what you see in Firefox and Chrome.

**Reply by STaylor135** [reply]
Thank you @onemen , 

I will report the bug in the proper place.  Here is an image of the issue, the non-badged browser is chorme 127.
Looks like it's just showing the alt text.  Don't look for the other report that soon.  A bit busy at work and I'm going to verify directly before making the report.  So I'll need some time to spin up a VM too. 

![TMP Instructions](https://github.com/user-attachments/assets/00d0f7e0-7907-4995-b746-799797cc28a2)

**Reply by onemen** [reply]
I have just tested Firefox 130 with the latest Firefox scripts and i work without any issue (Windows 11).
**Important:** when you updating Firefox scripts you must install new files in both places:
  - configuration files in your Firefox root directory
  - usilts files to each profile chrome folder.
  
Regarding images above, I don't see this issue on my Windows 11.
What is your operating system?
Try to reload with skipped cache Shift+F5 or Shift+R.

**Comment by RealEncounter** [comment]
Firtst,  nice to see that Tab Mix Plus is there again, I love this Tool, it is one of the important Things for FF

Second, the Problems Guys are Windows rights!
You have to allow all these Files from TMP with the right Mouse Button(Adminrights needed)
If you dont do this your FF says File is broken (but is isnt)

**Reply by pyrates999** [reply]
Elevated privileges to copy some of the files are a technical requirement here right now, sorry.
